### PR TITLE
Added steps to update configuratoin parameters

### PR DESCRIPTION
**Pull request by vnandwana**

### CODE REVIEW

File: userguide/tutorials/aws-cf.adoc [hunk 1]
```diff
@@ -119,6 +120,28 @@ Otherwise, you will see that your stack is being created, and its status (shown
...
+
+* Step 1 - Update stack: select *Use current template* and click Next
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step1.JPG[align=center]
+
+* Step 2 - Specify stack details: update the values of any of the parameters and click Next
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step2.JPG[align=center]
+
+* Step 3 - Configure stack options: no changes are needed on this step
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-cf.adoc]:
```suggestion
* Step 3 - Configure stack options: no changes are needed at this step
```

File: userguide/tutorials/aws-cf.adoc [hunk 2]
```diff
@@ -119,6 +120,28 @@
...
 If the create succeeds, the status will eventually change to CREATE_COMPLETE (shown in green). This may take a fairly long time.
 
+== Update Configuration
+To update the Kill Bill configuration parameters, select the CloudFormation stack and click on *Update*. The update stack consists of the following 4 steps:
+
+* Step 1 - Update stack: select *Use current template* and click Next
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step1.JPG[align=center]
+
+* Step 2 - Specify stack details: update the values of any of the parameters and click Next
```
[the remark this answers is not in the document]
Comment (reshmabidikar) [reply userguide/tutorials/aws-cf.adoc]: Does anything need to be done to get rid of this warning?

File: userguide/tutorials/aws-cf.adoc [hunk 3]
```diff
@@ -119,6 +120,28 @@ Otherwise, you will see that your stack is being created, and its status (shown
...
+* Step 3 - Configure stack options: no changes are needed on this step
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step3.JPG[align=center]
+
+* Step 4 - Review: review all the changes made to the parameters, scroll down and check *I acknowledge that AWS CloudFormation might create IAM resources* and click on *Submit*. 
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step4a.JPG[align=center]
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step4b.JPG[align=center]
+
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-cf.adoc:141]: Any particular reason for creating two separate screenshots here?

**Review by reshmabidikar — APPROVED** [review]
Left minor comments, good to merge once these are addressed.

File: userguide/tutorials/aws-cf.adoc (hunk 3 above)
Comment (reshmabidikar) [reply userguide/tutorials/aws-cf.adoc:141]: @vnandwana, is it necessary to have separate screenshots? The resultant image looks a bit off: 
image

Comment (vnandwana) [reply userguide/tutorials/aws-cf.adoc:141]: @reshmabidikar I agree, the first image is not really needed. I have made the necessary changes. Please let me know if you feel any more changes are needed. 
Appreciate your comments and feedback. Thank you.

Comment (reshmabidikar) [reply userguide/tutorials/aws-cf.adoc:141]: Looks good now, thanks!

**Pull request merged**